client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -59,6 +53,33 @@ const enableImageEditing = ( item, site ) => {
 	return true;
 };
 
+/**
+ * This function return true if the video editor can be
+ * enabled/shown
+ *
+ * @param  {object} item - media item
+ * @param  {object} site - current site
+ * @return {boolean} `true` if the video editor can be enabled.
+ */
+const enableVideoEditing = ( item, site ) => {
+	// do not show if the feature flag isn't set
+	if ( ! config.isEnabled( 'post-editor/video-editor' ) ) {
+		return false;
+	}
+
+	// do not allow for Jetpack site with VideoPress disabled
+	if ( get( site, 'jetpack', false ) ) {
```

Comment:
note that `get()` already returns `undefined` if the property doesn't exist which would be falsey here anyway

what is the goal? is it testing for the presence of the property (something `has()` does)? or is it testing if the property is `true`? sometimes I like to consider deviating from my norm when the context isn't clear enough

```js
if ( true === get( site, 'jetpack' ) )
```

but it's hard to tell what the true-side values would look like here